### PR TITLE
Correct DHI in era5 weather data

### LINKED ISSUE #193: update to new calculation of dhi and dni
after the discussion with INS the calculation of dhi should be adapted to:

weather_df["dhi"] = weather_df["ghi"] - (weather_df["dni"] * np.cos(np.deg2rad(spa["zenith"])))

### PR DESCRIPTION
**Pull request by Piranias**
Fix  #193

**Changes proposed in this pull request**:
- weather_df["dhi"] = weather_df["ghi"] - (weather_df["dni"] * np.cos(np.deg2rad(spa["zenith"])))

Notice: After merging this pull request everyone needs to recalculate their weather data. I could as well upload two locations (berlin, madrid) to static_inputs, if wanted)

The following steps were realized, as well (if applies):
-  :x: Use in-line comments to explain your code
-  :x: Write docstrings to your code
-  :x: For new functionalities: Explain in readthedocs
-  :x: Write test(s) for your new patch of code
- :x: Update the CHANGELOG.md (let's start this after the first release)
- [x] Apply black (`black . --exclude docs/`)

*Please mark above checkboxes as following:*
- [ ] Open
- [x] Done

:x: Check not applicable to this PR

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl-institut/mvs_eland/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

File: pvcompare/era5.py [hunk 1]
```diff
@@ -146,19 +150,19 @@ def format_pvcompare(ds):
     # convert temperature to Celsius (from Kelvin)
     ds["temp_air"] = ds.t2m - 273.15
 
-    ds["dirhi"] = (ds.fdir / 3600.0).assign_attrs(units="W/m^2")
+    # ds["dirhi"] = (ds.fdir / 3600.0).assign_attrs(units="W/m^2")
```
Comment (SabineHaas) [inline pvcompare/era5.py:153]: Why do you remove "dirhi"?

File: pvcompare/main.py [hunk 2]
```diff
@@ -152,7 +152,7 @@ def apply_pvcompare(
         user_inputs_pvcompare_directory=user_inputs_pvcompare_directory,
         user_inputs_mvs_directory=user_inputs_mvs_directory,
         year=year,
-        normalization="NSTC",
+        normalization=None,
```
Comment (SabineHaas) [inline pvcompare/main.py:155]: Should be False, right?, but was changed in another PR, not yet in dev. So, you could remove the changes here.

**Review by SabineHaas — APPROVED** [review]
Nice :+1: I'm so glad that CPV time series match Insolight's expectations now.

File: pvcompare/era5.py (hunk 1 above)
Comment (Piranias) [reply pvcompare/era5.py:153]: It was used to calculate the dhi before. But nice dhi is now calculated with ghi and dni, this is not needed anymore.

before:
    # ds["dhi"] = (ds.ghi - ds.dirhi).assign_attrs(
    #    units="W/m^2", long_name="direct irradiation"
    # )

**Comment by SabineHaas** [comment]
> Notice: After merging this pull request everyone needs to recalculate their weather data. I could as well upload two locations (berlin, madrid) to static_inputs, if wanted)

@Piranias @MaGering  I think an upload of data for the two locations would be great - also for the examples..?

**Comment by Piranias** [comment]
I will put two data sets (germany, spain) in static_inputs. So it's available in examples as well. see PR #204

**Pull request merged**